pages/docs/concepts/asyncapi-document/add-server.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Add servers
+weight: 295
+---
+
+A server represents a message broker handling communication between producers and consumers. Adding and defining servers is useful because it specifies where and how to connect. The connection facilitates where to send and receive messages.
+
+A unique key identifies the server and contains information about the server's connection details, such as the URL, protocol, and authentication. The diagram below describes how to define and add servers in AsyncAPI.
+
+```mermaid
+graph TD
+  subgraph AsyncAPI
+    A[Channel: userActions]
+    B[Operation: subscribe]
+    C[Message]
+    D[Payload]
+  end
+
+  subgraph Servers
+    SA[Servers: development, production]
+  end
+
+style A fill:#47BCEE,stroke:#47BCEE;
+style B fill:#47BCEE,stroke:#47BCEE;
+style C fill:#47BCEE,stroke:#47BCEE;
+style SA fill:#47BCEE,stroke:#47BCEE;
+style D fill:#47BCEE,stroke:#47BCEE;
+
+  SA --> A
+
+  A -->|inherits servers| B
+  B -->|references| C
+  C -->|references| D
+```
+
+For more information, refer [introduction to servers](/docs/server.md)
+
+## Define Servers
```

Comment:
```suggestion
## Define servers
```

We observe sentence capitalization as you see here in my commit suggestion fix, please go ahead and now fix the rest of your headers. ✌🏽